### PR TITLE
dev2 - minor changes

### PR DESCRIPTION
**Pull request by proddy**
get standalone working with espMqttClient

### CODE REVIEW

**Review by MichaelDvP — APPROVED** [review]
On my computer standalone does not compile, but this was always the case. Don't know what is wrong.

File: lib/framework/MqttSettingsService.cpp [hunk 1]
```diff
@@ -212,6 +212,8 @@ bool MqttSettingsService::configureMqtt() {
         static_cast<espMqttClient *>(_mqttClient)->setCleanSession(_state.cleanSession);
         return _mqttClient->connect();
     }
+    emsesp::EMSESP::logger().info("Connecting to MQTT broker...");
```
[the remark this answers is not in the document]
Comment (proddy) [reply lib/framework/MqttSettingsService.cpp:215]: yes, realized that when I didn't see the log. I'll just remove the line.

File: platformio.ini [hunk 2]
```diff
@@ -2,10 +2,10 @@
 ; override any settings with your own local ones in pio_local.ini
 
 [platformio]
-default_envs = esp32_4M
+; default_envs = esp32_4M
 ; default_envs = lolin_s3
 ; default_envs = esp32_16M
-; default_envs = standalone
+default_envs = standalone
```
[the remark this answers is not in the document]
Comment (proddy) [reply platformio.ini:8]: nope, mistake and was fixed

**Comment by proddy** [comment]
> On my computer standalone does not compile, but this was always the case. Don't know what is wrong.

I'm using Ubuntu WSL on Windows 10. I'll try native Windows and see what is happening.

**Comment by proddy** [comment]
> > On my computer standalone does not compile, but this was always the case. Don't know what is wrong.
> 
> I'm using Ubuntu WSL on Windows 10. I'll try native Windows and see what is happening.

I followed https://docs.platformio.org/en/latest/platforms/native.html and installed MSYS2 to get g++ and modified the paths. And yes, it doesn't compile and it would take too much work getting the standalone libs to work this way. It also takes more than 3mins to build, compared to 40 seconds on WSL on the same machine. So sticking to Linux and using WSL on Windows for now.

**Review by MichaelDvP — APPROVED** [review]

**Pull request merged**